cfme/tests/cloud/test_providers.py

```diff
@@ -1549,3 +1562,48 @@ def test_provider_compare_ec2_provider_and_backup_regions(appliance):
 
     assert regions_provider_texts == regions_scheduled_backup_texts
     assert regions_provider_texts == regions_immediate_backup_texts
+
+
+@test_requirements.cloud
+@pytest.mark.meta(automates=[1632750], blockers=[BZ(1632750,
+                                                 unblock=lambda child_provider: "object_managers"
+                                                 in child_provider)])
+@pytest.mark.uncollectif(lambda child_provider, provider:
+                        (provider.one_of(EC2Provider) and "object_managers" in child_provider) or
+                        (provider.one_of(AzureProvider) and ("object_managers" or "block_managers"
```

Comment:
Consider inclusive instead of exclusive - `provider.one_of(AzureProvider) and (child_provider != 'network_providers')`

Reply:
Reworked